tests/base/src/test/java/org/wildfly/security/auth/realm/DistributedSecurityRealmTest.java

```diff
@@ -68,6 +71,13 @@ public void setup() throws Exception {
         realm2 = createRealmWithIdentity("user2", createPasswordCredential(pass2));
         realm3 = createRealmWithIdentity("user3", createPasswordCredential(pass3));
         evidenceRealm = new SimpleEvidenceRealm();
+        ldapRealm = LdapSecurityRealmBuilder.builder()
```

Comment:
> ldapRealm

Something like `unavailableRealm` would be more descriptive.

Reply:
Thanks @OndrejKotek, updated.